include/AsyncLogger/Logger.hpp

```diff
@@ -1,5 +1,5 @@
 #pragma once
-#if !defined(USE_FMT) && CXX_FORMAT_SUPPORT
+#if !defined(USE_FMT) && CXX_FORMAT_SUPPORT || true

```

Comment:
Such hack not needed when add_compile_definitions(CXX_FORMAT_SUPPORT) is properly defined